react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/App.js

```diff
@@ -22,13 +22,22 @@ import "./css/App.css";
 import { Login } from "./login/login.js";
 import { ResponseError } from "./errorPage/index.js";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { logout } from "./login/logout.js";
+import { User } from "./data/User";
+
+const idToken = User.idToken;
+
+    const handleLogout = () => {
+      logout(idToken); 
+  };
 
 export const App = () => {
   return (
     <Router>
       <Switch>
         <Route path="/consentmgr" exact component={Login} />
         <Route path="/consentmgr/error" exact component={ResponseError} />
+        <Route path="/consentmgr/logout" exact component={handleLogout} />
```

Comment:
```suggestion
        <Route path="/consentmgr/logout" exact component={() => logout(User?.idToken)} />
```

Reply:
fixed in e3f02ac73e2598e97bbef38524ad149927667e94